kubernetes/helm/templates/deployment.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nebula-graphd
+  name: nebula-graphd
+  namespace: {{ .Values.namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: nebula-graphd
+  replicas: {{ .Values.replication.graphd.replicas }}
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate  
+  template:
+    metadata:
+      labels:
+        app: nebula-graphd
+    spec:
+      restartPolicy: Always
+{{- if .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 8 }}
+{{- end }}
+      containers:
+      - name: nebula-graphd
+        image: "{{ .Values.image.graphd.repository }}:{{ .Values.image.graphd.tag }}"
+        imagePullPolicy: {{ .Values.image.graphd.pullPolicy }}
+        resources:
+          requests:
+            cpu: {{ .Values.resources.graphd.requests.cpu | quote }}
+            memory: {{ .Values.resources.graphd.requests.memory | quote }}
+          limits:
+            cpu: {{ .Values.resources.graphd.limits.cpu | quote }}
+            memory: {{ .Values.resources.graphd.limits.memory | quote }}
+        ports:
+        - name: thrift
+          containerPort: {{ .Values.port.graphd.thriftPort }}
+        - name: http
+          containerPort: {{ .Values.port.graphd.httpPort }}
+        - name: http2
+          containerPort: {{ .Values.port.graphd.http2Port }}
+        command: ["/usr/local/nebula/bin/nebula-graphd", "--flagfile=/usr/local/nebula/etc/nebula-graphd.conf"]     
```

Comment:
Please  refer this entrypoint: https://github.com/vesoft-inc/nebula/blob/3208cabc9320674e04c283e9028c3a4e4b499cde/docker/Dockerfile.graphd#L21